lib/components/Portal/PortalProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {
+	Context,
+	createContext,
+	FunctionComponent,
+	RefObject,
+	useContext,
+	useRef,
+} from 'react';
+
+interface Props {
```

Comment:
This interface can go too, FunctionComponent gives you children by default.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e32d4f0292c57f63310af65a7fb3708155cf8083/types/react/index.d.ts#L517-L523